frontend-react/src/components/CreateEventForm/CreateEventForm.tsx

```diff
@@ -117,59 +134,66 @@ export const CreateEventForm = ({onSubmit, owner}: CreateEventFormProps) => {
                                     }),
                                 ]}
                             >
-                                <DatePicker.RangePicker style={{width: '100%'}} format="eeee, MMMM d" minuteStep={15}/>
+                                <DatePicker.RangePicker
+                                    style={{ width: '100%' }}
+                                    format="eeee, MMMM d"
+                                    minuteStep={15}
+                                />
                             </Form.Item>
                         </Col>
                     </Row>
                     <Row gutter={16} justify="center">
                         <Col span={16}>
                             <Form.List name="guests" initialValue={[{email: '', obligatory: true}]}>
-                                {(fields, {add, remove}) => (
-                                    <>
+                                {(fields, { add, remove }) => (
+                                    <div>
                                         {fields.length === 0 && <Typography.Text>No guests added yet.</Typography.Text>}
-                                        <div style={{height: '25vh', overflowY: 'auto'}}>
+                                        <div style={{flex: 1, overflowY: 'hidden', height: '160px' }}>
```

Comment:
Why are we going back to 'px'? Didn't we agree to convert everything to values relative to viewport (vh, vw)?

Reply:
I added it to make it look good for now. In later fixes we need to convert everything to vh

Reply:
why not do it now?

Reply:
because we will forget

Reply:
1) I want main to be working ASAP
2) I want to do it in a separate task to make it easier to review, I'll add it to the board to not forget

Reply:
ok then, but i feel like then this wont get done